src/sentry/api/endpoints/organization_release_details.py

```diff
@@ -186,6 +189,14 @@ def put(self, request, organization, version):
                         datetime=release.date_released,
                     )
 
+            analytics.record(
+                "release.set_commits",
+                user_id=request.user.id if request.user and request.user.id else None,
```

Comment:
@NisanthanNanthakumar I don't think it's possible for `request.user` to be missing. Every authenticated request should be tied to a user.